src/ASIFT_Master_Deliverable_1.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 20 12:24:43 2018
+
+ASIFT_Master_Deliverable_1.py -- Runs the master algorithm, using each phase code as needed.
+for Deliverable 1, take two images, run ASIFT against each of them, and generate matching keypoints as a CSV file.
+If the target image is geo-located, output the geo-location (lat/lon) of each keypoint as well.
+
+@author: mmacferrin
+"""
+import argparse
+
+# TODO: UPDATE THESE IMPORTS TO REFLECT LATEST CODE CHANGES.
+# Import sub-directory phases
+import PHASE_1_IMAGE_PREPROCESSING.ASIFT_Tile_Images as TILING
+import PHASE_2_KEYPOINT_GENERATION.ASIFT_Scheduler as ASIFT
+import PHASE_3_RANSAC_FILTERING.ASIFT_RANSAC_filter as RANSAC
+
+
+def process_two_single_images_for_keypoints(source_image, target_image, output_CSV, verbose=True):
```

Comment:
Agreed that "process_two_images" is a better name for it. I hadn't touched that code in awhile, and indeed perhaps it could be dropped entirely for now, and replaced with a "process_two_images.py" code shortly.

Reply:
Just to add, a rapidly growing number of the “todo” issues you’re bringing up have little to do with merging the branch. Things like implementing CSV read/writes with Pandas, and getting rid of the “eprint” function, etc. They aren’t points I disagree with but are unecessary in this pull request, are more just modification suggestions to be done after a merge. Honestly it’d help a bit if the thread were actually focused on the pull request itself.

Close the extraneous conversations and we can focus on what actually needs doing in the pull request.